delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/model/StorageRejection.scala

```diff
@@ -266,4 +269,21 @@ object StorageRejection {
 
   implicit final val evaluationErrorMapper: Mapper[EvaluationError, StorageRejection] = StorageEvaluationError.apply
 
+  implicit final val storageRejectionHttpResponseFields: HttpResponseFields[StorageRejection] =
```

Comment:
If you are putting the `HttpResponseFields` in the *Rejection instead of in the *Routes, can you do it as well for `ElasticSearchViews` and `BlazegraphViews` to be consistent?

Reply:
right, done